include/swift/Frontend/FrontendOptions.h

```diff
@@ -85,6 +85,9 @@ class FrontendOptions {
   /// The module for which we should verify all of the generic signatures.
   std::string VerifyGenericSignaturesInModule;
 
+  /// We should only include public symbols in the objc compatibility header.
+  bool ObjcCompatibilityHeaderIsPublic = false;
```

Comment:
Nit: Here and elsewhere, change `ObjcCompatibility` to `ObjC`. This will make it consistent with terminology elsewhere in the compiler, like `ObjCHeaderOutputPath`, so it'll be more clear that they're used for the same purpose.